### ISSUE TITLE
I want to enable SSH key authentication on MS Azure Ubuntu VM

### ISSUE THREAD

**Issue by muratagenc** [dps2018-dcs801/t1-bank#2]
using ssh-keygen or Putty to generate public and private keys for MS Azure authentication

**Comment by cnjelita** [comment]
I thought we are not required to generate public and private keys for MS Azure authentication  according to Dr Rinaldo. I created username and password for my install and authentication exercise we through the last session. He said we may stop there until next time we meet with him. Anybody, know when will have that session?

**Comment by muratagenc** [comment]
no, it's not a requirement, it was an opportunity to learn it. I took the liberty.
